pkg/scheduler/scheduler.go

```diff
@@ -139,6 +139,7 @@ type ScheduleResult struct {
 	SuggestedHost string
 	// The number of nodes the scheduler evaluated the pod against in the filtering
 	// phase and beyond.
+	// Note that it contains the number of nodes that filtered out by PreFilterResult.
 	EvaluatedNodes int
```

Comment:
EvaluatedNodes is only used in logging, not being used at anywhere else like any functionalities or metrics. So, this change doesn't cause anything bad.
https://github.com/kubernetes/kubernetes/blob/6cf77098aaf250697abc1d97fc9e576903b95868/pkg/scheduler/schedule_one.go#L304
(It leads us to another question; whether this `EvaluatedNodes` is worthwhile to keep or not in the first place though)

Reply:
I've find this information useful when debugging in the past.